nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestCommentNotification.java

```diff
@@ -103,8 +111,15 @@ protected CommentManager getCommentManager() {
     public void shouldNotifyEventWhenCreateComment() {
         // We subscribe to the creation document to check that we will not be notified about the comment creation as
         // document (see CommentCreationVeto), only the comment added, and the 'File' document creation
+        DocumentModel comment = captureAndVerifyCommentEventNotification(() -> {
+            Comment createdComment = createCommentAndAddSubscription(commentedDocumentModel, "CommentAdded", "Creation");
+            return session.getDocument(new IdRef(createdComment.getId()));
+        }, COMMENT_ADDED, DOCUMENT_CREATED);
+
+        commentedCommentDocumentModel = comment;
+
         captureAndVerifyCommentEventNotification(() -> {
-            Comment createdComment = createCommentAndAddSubscription("CommentAdded", "Creation");
+            Comment createdComment = createCommentAndAddSubscription(comment, "CommentAdded", "Creation");
```

Comment:
You should create your own test.

Reply:
I did so, but this duplicates a lot of code. I would rather have kept the extra argument.